src/Entity/AbstractValuation.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+namespace Jedkirby\ValPal\Entity;
+
+abstract class AbstractValuation
+{
+    /**
+     * @var string
+     */
+    protected $minValuation;
+
+    /**
+     * @var string
+     */
+    protected $valuation;
+
+    /**
+     * @var string
+     */
+    protected $maxValuation;
+
+    /**
+     * @var string
+     */
+    protected $propertyType;
+
+    /**
+     * @var string
+     */
+    protected $tenure;
+
+    /**
+     * @var string
+     */
+    protected $bedrooms;
+
+    /**
+     * @var string
+     */
+    protected $propertyConstructionYear;
+
+    /**
+     * @var string
+     */
+    protected $minRentalValuation;
+
+    /**
+     * @var string
+     */
+    protected $rentalValuation;
+
+    /**
+     * @var string
+     */
+    protected $maxRentalValuation;
+
+    /**
+     * @param string $value
+     */
+    public function setMinValuation($value)
+    {
+        $this->minValuation = $value;
```

Comment:
just checking, but assuming this is a string as it allows for descriptive valuations, and not just floats?